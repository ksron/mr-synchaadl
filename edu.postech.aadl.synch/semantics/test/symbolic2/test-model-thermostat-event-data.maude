
set show timing off .

load ../../interpreter-symbolic2.maude
load ../syntax/module-thermostat-event-data.maude


mod TEST-THERMOSTAT-SYSTEM-MOD is
  including THERMOSTAT-SYSTEM-SYNTAX .
  including BEHAVIOR-SYMBOLIC-LOCATION . 
  including MODEL-TRANSITION-SYSTEM .
  including SMT-REAL-EXP .
  
  eq init  = loc(real(0)) .
  eq state = loc(real(1)) .

  op initState : -> Object .
  eq initState = initialize(model) .
endm


search [,5] {bool(true) || initState | 0 | infty} =>+ C:GlobalSystem .
