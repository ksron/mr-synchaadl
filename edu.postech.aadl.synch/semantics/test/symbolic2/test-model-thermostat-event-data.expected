==========================================
search [, 5] in TEST-THERMOSTAT-SYSTEM-MOD : {bool(true) || initState | 0 |
    infty} =>+ C:GlobalSystem .

No solution.
states: 1  rewrites: 461
Bye.
